### PR TITLE
Add hack/create-bundle

### PR DESCRIPTION
**Pull request by jsafrane**
To create + push bundle image + index image for local testing.

### CODE REVIEW

**Comment by gnufied** [comment]
Tested it and it appears to work.

File: README.md [hunk 1]
```diff
@@ -33,3 +33,15 @@ export OPERATOR_NAME=aws-efs-csi-driver-operator
...
 ```
+
+
+# OLM
+
+To build an bundle + index images, use `hack/create-bundle`.
+
+```shell
+cd hack
+./create-bundle quay.io/openshift/origin-aws-efs-csi-driver:latest quay.io/openshift/origin-aws-efs-csi-driver-operator:latest quay.io/<my-repo>/efs-bundle quay.io/<my-repo>/efs-index
```
Comment (gnufied) [inline README.md]: nit: can you add a note that, these images aren't mirrored yet?

Comment (jsafrane) [reply README.md]: Updated the example to CI registry with a TODO to replace it with quay once we have the images.

**Pull request merged**